docs/pages/attributesJSON.rst

```diff
@@ -0,0 +1,194 @@
+Using JSON Files to configure Attributes
+########################################
+
+:ref-prefix:
+    habitat_sim.sim
+    habitat_sim.attributes
+    habitat_sim.attributes_managers
+
+:summary: This document describes the appropriate tags to be used when authoring JSON files to properly customize attributes templates.
+
+.. contents::
+    :class: m-block m-default
+
+Attributes templates provide a mechanism by which the various constructions in Habitat can be customized and built with user-specified characteristics, either at program start or on the fly.
+
+`Physics Manager Attributes`_
+=============================
+Physics Manager Attributes templates describe quantities pertinent to building the simulation world.  Any source configuration JSON files used to build these attributes should be named in the following format :
+
+ 	<worldname>.phys_scene_config.json
+
+An example of an appropriate configured file can be found below :
+
+.. include:: ../../data/test_assets/testing.phys_scene_config.json
+    :code: json
```

Comment:
May want to link/name the file you are referencing here so it can be found by the user.

Reply:
Made links.